akka-actor/src/main/scala/akka/actor/Scheduler.scala

```diff
@@ -44,8 +44,9 @@ trait Scheduler {
     initialDelay: FiniteDuration,
     interval: FiniteDuration,
     receiver: ActorRef,
-    message: Any)(implicit executor: ExecutionContext,
-                  sender: ActorRef = Actor.noSender): Cancellable =
+    message: Any)(implicit
+    executor: ExecutionContext,
+    sender: ActorRef = Actor.noSender): Cancellable =
```

Comment:
This looks ugly. Is it possible to keep the remaining parameter lists on the same line as it otherwise kinda hides them?
```scala
final def schedule(
    initialDelay: FiniteDuration,
    interval: FiniteDuration,
    receiver: ActorRef,
    message: Any)(implicit executor: ExecutionContext, sender: ActorRef = Actor.noSender): Cancellable =
```